vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentDetailsComponent.java

```diff
@@ -129,20 +139,19 @@ private DisclaimerCard createNoVariableDisclaimer() {
   }
 
   private void layoutComponent() {
+    this.add(header);
+    header.addClassName("header");
     this.add(content);
+    //Necessary to avoid css collution
```

Comment:
?

Reply:
As we discussed there is an issue that the css from the .content class of the experiment-details-component interacts with the .content css class of the experimental-groups-collection and the .content of the experimental-variable. Since this seems to be a bigger issue i proposed a small fix for this PR